user/test/com/google/gwt/emultest/java/math/BigIntegerConstructorsTest.java

```diff
@@ -605,6 +606,62 @@ public void testConstructorSignBytesZeroNull3() {
     assertEquals("incorrect sign", 0, aNumber.signum());
   }
 
+  @Test
```

Comment:
This annotation isn't required/necessary, all tests are Junit3 :(.

Could you add tests that confirm working behavior, in addition to these negative test cases? It is probably enough to take a few tests above that use the old constructor and just pad them out with other numbers (0 or 127 so we have all bits and no bits set), and set offset/length to confirm that only the correct values are used?